### PR TITLE
Add dockerfile to build kafka and yaml file for deploy

### PR DESCRIPTION
**Pull request by huangyuqi**
+ add dockerfile and script to build docker image includes kafka and zookeeper 

```
├─kafka
│  │  build.sh
│  │  Dockerfile
│  │  README.md
│  │
│  ├─scripts
│  │      start-kafka.sh
│  │
│  └─supervisor
│          kafka.conf
│          zookeeper.conf
```

+ add deploy file to run a kafka & heapster instance in PODs

```
└─kube-config
   ├─kafka
   │      heapster-controller.yaml
   │      heapster-service.yaml
   │      kafka-controller.yaml
   │      kafka-service.yaml
```

### CODE REVIEW

File: deploy/kube-config/kafka/heapster-service.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```
Comment (vishh) [inline deploy/kube-config/kafka/heapster-service.yaml]: Can we include all the configs in a single file?

File: kafka/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Kafka with Zookeeper for kubernetes' heapster
```
Comment (vishh) [inline kafka/Dockerfile]: What's special about this image? Is there an existing image that we can use?

File: kafka/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+heapster_kafka
+================
+
+Monitoring and alerting for Kubernetes container metrics
```
Comment (vishh) [inline kafka/README.md]: How is alerting possible?

File: kafka/Dockerfile (hunk 2 above)
Comment (huangyuqi) [inline kafka/Dockerfile]: nothing special about this image.
we can find a similar image in docker hub.

File: deploy/kube-config/kafka/heapster-service.yaml (hunk 1 above)
Comment (huangyuqi) [inline deploy/kube-config/kafka/heapster-service.yaml]: Yes, the `yaml` file supports  delimiter `---` to define a block.
So we can put all configs in a single file, separate with `---`.Like:

```
---
apiVersion: v1
kind: ReplicationController
metadata:
  name: kafka
 ....

---
apiVersion: v1
kind: Service
metadata:
  labels:

```

File: kafka/README.md (hunk 3 above)
Comment (huangyuqi) [inline kafka/README.md]: this is a typo error. i will remove. Thanks.

File: deploy/kube-config/kafka/heapster-service.yaml (hunk 1 above)
Comment (huangyuqi) [inline deploy/kube-config/kafka/heapster-service.yaml]: Is putting all config in a single file not clearness enough?

**Comment by mwielgus** [comment]
LGTM

**Pull request merged**